### PR TITLE
Update appstream data on "flatpak update"

### PR DESCRIPTION
**Pull request by alexlarsson**
This is especially useful since we now have flatpak search which uses
this data.

### CODE REVIEW

File: doc/flatpak-update.xml [hunk 1]
```diff
@@ -73,6 +73,10 @@
             Unless overridden with the --user or the --installation option, this command updates
             the default system-wide installation.
         </para>
+        <para>
+            If no REF is given, then all installed apps and runtimes in the installation is updates, as
```
Comment (mwleeds) [inline doc/flatpak-update.xml]: should say "are updated"

File: doc/flatpak-update.xml [hunk 2]
```diff
@@ -73,6 +73,10 @@
...
+            well as appstream info for all remotes..
```
Comment (mwleeds) [inline doc/flatpak-update.xml]: extra period

**Comment by alexlarsson** [comment]
Found a better place to document the change.

**Comment by mwleeds** [comment]
Forgot to use @rh-atomic-bot, sorry

**Pull request merged**